modules/mono/glue/GodotSharp/GodotSharp/Core/NativeInterop/NativeFuncs.cs

```diff
@@ -37,6 +37,8 @@ private partial struct UnmanagedCallbacks
 
         // Custom functions
 
+        internal static partial bool godotsharp_dotnet_module_is_initialized();
```

Comment:
```suggestion
        internal static partial godot_bool godotsharp_dotnet_module_is_initialized();
```

We had that one a few days ago, lets not reintroduce that type of crash.